model2vec/model.py

```diff
@@ -45,14 +49,75 @@ def __init__(self, vectors: np.ndarray, tokenizer: Tokenizer, config: dict[str,
             self.unk_token_id = None
 
         self.config = config
+        if normalize is not None:
+            self.normalize = normalize
+        else:
+            self.normalize = config.get("normalize", False)
+
+    @property
+    def normalize(self) -> bool:
+        """
+        Get the normalize value.
+
+        :return: The normalize value.
+        """
+        return self._normalize
+
+    @normalize.setter
+    def normalize(self, value: bool) -> None:
+        """Update the config if the value of normalize changes."""
+        config_normalize = self.config.get("normalize", False)
+        self._normalize = value
+        if config_normalize is not None and value != config_normalize:
```

Comment:
```suggestion
        if value != config_normalize:
```
The check for None is not needed right, since the default value for the get in line 69 is False?